### ISSUE TITLE
Segfaults when input is piped from non-existant program

### ISSUE THREAD

**Issue by youRFate** [bartobri/no-more-secrets#6]
When I call a command that does not exist and pipe it's non existant output into nms it will segfault. Sample output:

```
zsh/4 1002 [4] % nonexistant | nms
zsh: command not found: nonexistant
zsh: exit 127                          nonexistant | 
                                                     zsh: segmentation fault (core dumped)  nms
                                                                                               %
```
Afterwards zsh behaves weirdly, indenting some output seemingly at random.

Some info:
x86_64
arch linux, 
gcc 5.3.0,
zsh 5.2,
ncurses 6.0

**Comment by bartobri** [comment]
Good catch! Looking in to this. Hopefully should have a fix soon.

**Comment by bartobri** [comment]
This issue is fixed now via commit b34fc9bf.
